### ISSUE TITLE
Adding conditional tfoot triggers table-groups validation

### ISSUE THREAD

**Issue by alanlalonde** [ember-template-lint/ember-template-lint#717]
I've built a table with a thead, tbody and conditional tfoot that seems to break the table-groups rule. Here is some sample code.

```handlebars
<table>
  <thead>
    <tr>
      <th>Header</th>
    </tr>
  </thead>
  <tbody>
    {{#each rows as |row|}}
      {{this-component
        row=row
      }}
    {{/each}}
  </tbody>
  {{#if trueOrFalse}}
    <tfoot>
      <tr>
        <td>Footer Content</td>
      </tr>
    </tfoot>
  {{/if}}
</table>
```

Any idea what would cause this to occur? thead and tbody are known to be required by this rule and are already in the table, so I don't know if this is either an edge case or a false positive.

**Comment by lifeart** [comment]
@alanlalonde can't reproduce it in latest master, could you add failing test case into `test\unit\rules\lint-table-groups-test.js`?

```js
    '<table>' +
      '<thead><tr><th>Header</th></tr></thead>' +
      '<tbody>{{#each rows as |row|}}{{this-component row=row}}{{/each}}</tbody>' +
      '{{#if trueOrFalse}}<tfoot><tr><td>Footer Content</td></tr></tfoot>{{/if}}' +
      '</table>',
```
as good example - working for me

**Comment by rwjblue** [comment]
Going to close for now, I _believe_ this was fixed by https://github.com/ember-template-lint/ember-template-lint/pull/791.
